waiter/integration/waiter/token_request_test.clj

```diff
@@ -410,9 +410,10 @@
         (try
           (log/info "basic hostname as token test")
           (let [current-user (retrieve-username)
-                service-description (assoc (kitchen-request-headers)
-                                      :x-waiter-permitted-user "*"
-                                      :x-waiter-run-as-user current-user)]
+                service-description (-> (kitchen-request-headers)
+                                        (dissoc :x-waiter-metric-group)
```

Comment:
Why is this `dissoc` necessary?

Reply:
Changed it to `assoc :x-waiter-metric-group token-prefix` as per our discussion.